Blackjack/Game/BlackjackGame.m

```diff
@@ -0,0 +1,254 @@
+//
+//  BlackjackGame.m
+//  Blackjack
+//
+//  Created by Alex on 28/09/2022.
+//
+
+#import "BlackjackGame.h"
+#import "PlayingCard.h"
+
+typedef NS_ENUM(NSInteger, State) {
+    COLLECT_BETS, DEAL_CARDS, AWAITING_PLAYERS_DECISION, AWAITING_DEALER
+};
+
+// MARK: - Private Properties
+
+@interface BlackjackGame ()
+
+@property (nonatomic, strong) Deck *deck;
+@property (nonatomic) NSInteger cardsAmountToWin;
+@property (nonatomic) NSInteger dealerMinimumCardEvaluation;
+@property (nonatomic, assign) State gameState;
```

Comment:
A parameter that you do not do anything with, and that is a shame...
When you progress your game state, you need to inform your delegate that your gameState has changed and the UI should be updated accordingly.

Right now, this parameter has no meaning

Reply:
Sounds right.